src/components/settings-row/settings-row.test.js

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import SettingsRow from "./settings-row.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import { getDataElementByValue } from "../../../cypress/locators";
+import {
+  settingsRowPreview,
+  settingsRowChildren,
+  settingsRowDescription,
+} from "../../../cypress/locators/settings-row/index";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+context("Tests for SettingsRow component", () => {
+  describe("should check SettingsRow component properties", () => {
+    it.each(testData)(
+      "should check %s as title for SettingsRow component",
+      (title) => {
+        CypressMountWithProviders(<SettingsRow title={title} />);
+        getDataElementByValue("title").should("have.text", title);
+      }
+    );
+
+    it.each(testData)(
```

Comment:
Add another test case for a description that is a non-text object, e.g. a Link, as this prop accepts any object.